packages/client/src/components/Location/LocationInput.tsx

```diff
@@ -137,7 +146,7 @@ const LocationInput: React.FC<LocationInputProps & MatomoTrackEventProps> = ({
           }}
         />
         <Nav
-          nextText={hasIMTR ? "Naar de vragen" : "Volgende"}
+          nextText={hasIMTR ? t("common.to the questions") : ("common.next")}
```

Comment:
I think you missed a `t` here.

Reply:
Great catch. still a draft though. Having a hard time fixing a location test.